src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -9,30 +9,32 @@
 			class="tw-mb-1 tw-rounded" :style="{width: '100%', height: '15.75rem'}"
 		/>
 
-		<!-- TODO: Once click interaction for this is in place, we need to disable the click
-		if allSharesReserved is true
-		ie.
-		:click="`${allSharesReserved ? 'correct redirect' : ''}`" -->
-		<borrower-image
-			v-if="!isLoading"
-			class="
+		<!-- Is there a way to display the click completely if allSharesAreReserved? -->
```

Comment:
You can use a v-if on the router-link and it's containing borrower-image then have the v-else on an duplicated instance of the image when you don't want to show a click...I do question why we would turn off the link at all in this scenario. I don't think we should be removing this link...